dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,24 +1,16 @@
 module SurveyResultsHelper
   def show_survey?
-    # rubocop:disable Lint/UnreachableCode
-
-    # Disabled for all users on 2016-03-30 after being shown for thirty days.
-    return false
-
     # Reasons we would not show the survey.
     unless current_user
       return false
     end
-    unless current_user.teacher?
```

Comment:
Fair point.  Right now I guess `show_survey?` just returns logic for whichever is the active survey, but it would be possible to parameterize it in future (or have variant `show_` functions) for multiple surveys.